TACC/Core-Portal
# Bug/WP-419 Public Data Header Left Margin

## Overview
Fixing the Public Data Header that was off alignment with the Public Data listing


## Related

* [WP-419](https://tacc-main.atlassian.net/browse/WP-419)
* Duplicated [WP-399](https://tacc-main.atlassian.net/browse/WP-399)

## Changes
- Passed in the contentLayoutName called "oneColumn" overriding the default contentLayoutName "hasSidebar". The contentLayoutName manages the spacing with and without the sidebar. (Hints from this came from how a Section component is created at SystemStatus.jsx:128)


## Testing
### Testing with access to Public Data Listings
1. Navigate to the Public Data Listings at either
https://cep.test/public-data
or
https://cep.test/public-data/tapis/public/cloud.data/corral/tacc/aci/CEP/public
3. Ensure that the Public Data Header is spaced left and right correctly

### If testing without access to Public Data
1. Inspect on the Public Data Header
2. Change the section class to 
```class="_root_1f30w_3 workbench-content workbench-wrapper c-section--has-content-layout-one-column"```
Additional screenshots in the UI will show the exact space where to edit this

### If testing in the codebase you can change the following at Section.jsx:119-122
To see the contentLayoutName translation.
```
  contentLayoutName = "oneColumn"
  console.log("The layout name is ", contentLayoutName)
  const layoutClass = getLayoutClass(contentLayoutName);
  console.log("The layout is ", layoutClass)
```
### To see other section content layouts
Navigate to SectionContent.jsx for a full list

## UI
### Before variable change
Layout name translation console output
```
Section.jsx:119 The layout name is  hasSidebar
Section.jsx:121 The layout is  c-section--has-content-layout-has-sidebar
```
![image](https://github.com/user-attachments/assets/b7a106f4-8ef2-4472-9200-19d19508022b)

### After variable change
Layout name translation console output
```
Section.jsx:120 The layout name is  oneColumn
Section.jsx:122 The layout is  c-section--has-content-layout-one-column
```
![image](https://github.com/user-attachments/assets/4875ee14-f73d-4705-9c50-b426cd8661e0)
